index.js

```diff
@@ -475,13 +475,13 @@ function Physics (mcData, world) {
     }
   }
 
-  function isMaterialInBB (world, queryBB, type) {
+  function isMaterialInBB (world, queryBB, ...types) {
```

Comment:
why the ... ?

Reply:
To be able to handle 1 or multiple types

Reply:
that does not work

if you give to a ...types a list, the argument will be a list of list

Reply:
Can you remove the ... and change all calls to pass a list ? It's simpler and better